cryptography

```diff
@@ -0,0 +1,15 @@
+---
+Title: 'Cryptography' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
```

Comment:
```suggestion
Title: 'Cryptography'
```

As you submit more PRs, I would expect this suggestion since the file looks cleaner, overall, without so many comments. They're there more as a guide for setting up.